### ISSUE TITLE
Undefined partials

### ISSUE THREAD

**Issue by excellent6** [dlmanning/gulp-sass#431]
I have a simple gulp task.
```javascript
    gulp.task('sass', function () {
        gulp.src('./source/main.scss')
           .pipe(sass())
           .pipe(gulp.dest('./dest');
    });
```
And this is my project structure:

 |_source
----|_sass
------ |_partials
---------_import.scss
-------main.scss
----|_dest
-------|_layout

## Main.scss
```sass
@import "partials/import";

body {color: orange}
````

## _import.scss
````sass
h6 {background-color: green)
````

When I run gulp, I get this error:
![Problem](http://i.imgur.com/vTxcLha.png)

My path is in <b>cyrillic</b>.

Any ideas.

**Comment by xzyfer** [comment]
There is a known issue cyrillic paths. This will be fixed in 3.5.0.

**Comment by agustin107** [comment]
Try to replace on package.json of gulp-sass, instead of
`"node-sass": "^3.4.2"`
with
`"node-sass": "^3.5.0-beta.1"`

This worked for me :+1:

**Comment by JeffWeb** [comment]
definitively working for me now, thx!
Hope node-sass beta version will come stable soon so it could be merged in project :)

**Comment by xzyfer** [comment]
This is fixed in v2.3.1.
